glusterd2-e2e-github-pr.yml

```diff
@@ -1,57 +1,54 @@
 - job:
-    name: 'gluster_glusterd2'
+    name: 'gluster_glusterd2_e2e'
     node: gluster
-    description: Verify GlusterD2 pull-requests
+    description: Runs e2e tests against glusterd2 pull requests
     project-type: freestyle
-
+    concurrent: true
 
     scm:
     - git:
-        branches:
-        - '*/master'
-        git-tool: Default
         url: https://github.com/gluster/glusterd2.git
+        branches:
+        - origin/master
     - git:
         url: https://github.com/gluster/centosci.git
         branches:
         - origin/master
 
-
     builders:
-    - shell: - shell: !include-raw scripts/common/jenkins-job.py
-
-    parameters:
-    - string:
-        default: scripts/common/run-test.sh
-        description: Test script to execute on the reserved machine.
-        name: TEST_SCRIPT
-
+    - shell: !include-raw: scripts/common/get-node.sh
+    - shell: !include-raw: scripts/common/rsync.sh
+    - shell: scripts/common/bootstrap.sh $WORKSPACE/extras/centos-ci.sh
 
     properties:
     - github:
         url: https://github.com/gluster/glusterd2/
-
+    - build-discarder:
+        days-to-keep: 7
+        artifact-days-to-keep: 7
 
     triggers:
     - github-pull-request:
         admin-list:
         - aravindavk
         - kshlm
         - prashanthpai
-        allow-whitelist-orgs-as-admins: false
+        org-list:
+        - gluster
         auth-id: a9df0596-2fe4-4e12-a8a5-6856d007bfbc
-        auto-close-on-fail: false
-        build-desc-template: null
         cron: H/5 * * * *
         github-hooks: true
-        only-trigger-phrase: false
-        org-list:
-        - gluster
-        permit-all: false
-        trigger-phrase: null
-        white-list-target-branches: []
+        trigger-phrase: retest this please
         cancel-builds-on-update: true
 
+    publishers:
+    # TODO: Export logs
```

Comment:
I'll send another PR on top of this to address this comment after confirming that the conversion from old format to JJB works.